firebase-tools peerDependency excludes v15 (the current major)

### Version info

**Angular:** 21.2.x (any Angular 21)

**Firebase:** 12.x

**AngularFire:** 21.0.0-rc.0 (`@next`)

**Other:** firebase-tools 15.x; npm / pnpm

### How to reproduce these conditions

**Steps to set up and reproduce**

1. Install `@angular/fire@next` in a project that also has `firebase-tools@15` (the current major).
2. The package manager emits an unmet/invalid optional-peer warning for `firebase-tools` (or an `ERESOLVE` under pnpm / `--strict-peer-deps`), because `@angular/fire` declares `"firebase-tools": "^14.0.0"`, which excludes 15.x.

This is a static package-metadata issue, reproducible directly from the published `@angular/fire` `package.json` — no Stackblitz needed.

### Debug output

```
npm warn ERESOLVE overriding peer dependency
npm warn Could not resolve dependency:
npm warn peerOptional firebase-tools@"^14.0.0" from @angular/fire@21.0.0-rc.0
```

### Expected behavior

The optional `firebase-tools` peer range includes the current firebase-tools major (15.x), so users on the current CLI get no spurious peer warning. The `ng add` runtime version check already only requires `>= 14.0.0`, and the schematic runs fine on 15.x.

### Actual behavior

`@angular/fire`'s `peerDependencies.firebase-tools` is `^14.0.0`, excluding 15.x. This peer range has repeatedly fallen behind the firebase-tools release train (see #3237, #3238, #3479). Proposed fix: `^14.0.0 || ^15.0.0`.

--- a/src/package.json
+++ b/src/package.json
@@ -29,7 +29,7 @@
     "@angular/platform-browser-dynamic": "^21.0.0",
     "@angular/platform-server": "^21.0.0",
     "rxjs": "~7.8.0",
-    "firebase-tools": "^14.0.0"
+    "firebase-tools": "^14.0.0 || ^15.0.0"
   },
   "peerDependenciesMeta": {
     "firebase-tools": { "optional": true },

